src/vs/workbench/contrib/search/browser/quickTextSearch/textSearchQuickAccess.ts

```diff
@@ -193,10 +196,35 @@ export class TextSearchQuickAccess extends PickerQuickAccessProvider<IPickerQuic
 					},
 				});
 			}
+		}
+		return picks;
+	}
+	protected _getPicks(contentPattern: string, disposables: DisposableStore, token: CancellationToken): FastAndSlowPicks<IQuickPickItem> {
```

Comment:
this function can actually return multiple things like:
* `FastAndSlowPicks`
* `Picks`

You should do this ..... and then if you only have sync results just return the picks.